sql/driver/driver_test.go

```diff
@@ -149,6 +149,9 @@ func TestCreateTable(t *testing.T) {
 	if _, err := db.Exec("CREATE TABLE IF NOT EXISTS users " + cols); err != nil {
 		t.Fatal(err)
 	}
+	if _, err := db.Exec("CREATE TABLE employees " + cols); err != nil {
```

Comment:
True. This just adds another command that doesn't fail.

BTW I looked at the driver interface and the way this code is currently structured, the session state will be local to a connection and will not span the entire DB client. If the DB uses more than one connection we will hit some odd behavior.